src/pepr/operator/controllers/network/generators/kubeNodes.ts

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright 2024 Defense Unicorns
+ * SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+ */
```

Comment:
General comment on this added file - would be great to add jest testing around these functions. We encountered a number of issues with the kubeAPI policy and added testing to cover those functions as a result, would be great to do similar here. You should be able to model after what was done for KubeAPI with Jest mocks to avoid any k8s calls, and most of that should be relatively easy to generate as well.

Reply:
Tests were added in 6921d23b